app/server/views/includes/contact.html

```diff
@@ -0,0 +1,17 @@
+<details class="govuk-details contact-include">
+  <summary class="govuk-details__summary">
+    <span class="govuk-details__summary-text">
+      {{ i18n.contact.summary }}
+    </span>
+  </summary>
+  <div class="govuk-details__text">
+    <h3>{{ i18n.contact.email.header }}</h3>
+    <p>
+      <a href="">{{ i18n.contact.email.link.text }}</a> {{ i18n.contact.email.link.newWindow }}<br />
+      {{ i18n.contact.email.text }}
+    </p>
+
+    <h3>{{ i18n.contact.telephone.header }}</h3>
+    <p>0300 123 1142<br />{{ i18n.contact.telephone.hours }}</p>
```

Comment:
Can we put this phone number in the `en.json` file?

Reply:
Ok